enhancements/etcd/automated-backups-no-config.md

```diff
@@ -0,0 +1,242 @@
+Using a `SideCar` container within each etcd pod, backups could be taken with no config from user.
+
+### User Stories
+
+* As a cluster administrator I would like OCP cluster backup to be taken without configuration.
+* As a cluster administrator I would like to schedule recurring OCP cluster backups so that I have a recent cluster state to recover from in the event of quorum loss (i.e. losing a majority of control-plane nodes).
+* As a cluster administrator I would like to have alerts upon failure to take OCP cluster backup.
+
+### API Extensions
+
+No [API](https://github.com/openshift/enhancements/blob/master/enhancements/etcd/automated-backups.md#api-extensions) changes are required, since this approach work independently with default config.
+
+### Topology Considerations
+TBD
+#### Hypershift / Hosted Control Planes
+TBD
+#### Standalone Clusters
+TBD
+#### Single-node Deployments or MicroShift
+TBD
+### Workflow Description
+TBD
+
+## Design Details
+
+A `SideCar` container within each etcd pod in Openshift cluster is being added in order to take backups without any provided configuration from user's perspective.
+
+In order to alleviate the overhead on etcd server, the side container takes the backup by simply copying the snapshot file.
+This approach is being used by `microshift`, and it has minimal overhead as it is a simple file system copy operation.
+Since the `SideCar` is being deployed alongside each etcd cluster member, it is possible to keep backups across all master nodes.
+
+On the other hand, the backups may **not** be up-to-date since the snapshot might be lagging behind the `WAL`. Therefore, it is recommended to use this approach alongside the Automated Backup enabled using the `EtcdBackup` CR.
+Since this work will be enabled with no configuration, it is possible to use define a default values for the `Scheule`, `Retention` independently.
+
+Since cluster backups must be stored on a persistent storage, and each storage solution has pros and cons, several approaches that have been investigated. See [Alternatives](#Alternatives)
+
+This work utilizes `hostPath` approach, since the `SideCar` container has access to the etcd pod's file system.
+
+### Open Questions
+
+This enhancement adds on previous work [automated backup of etcd](https://docs.openshift.com/container-platform/4.15/backup_and_restore/control_plane_backup_and_restore/backing-up-etcd.html#creating-automated-etcd-backups_backup-etcd).
+Therefore, it is vital to use the same feature gate for both approaches. The following issues need to be clarified before implementation
+
+* How to distinguish between `NoConfig` backups and configs that are triggered using `EtcdBackup` CR.
+* Upon enabling the `AutomatedBackup` feature gate, which approach should be used and according to what criteria.
+* How should the current backup controller distinguish between both approaches.
+* How to disable the `NoConfig` behaviour, if a user want to.
```

Comment:
that depends on how we install that "default" CRD, but it would either be a deletion of that CRD (so we need to ensure we don't install it back again) or an annotation that will disable it

Reply:
i will remove this since we do not install CR at all, it is a side car container that exist in all etcd pod manifests

Reply:
since there is no default CRD and we use the sidecar approach, we could 

- re-deploy the etcd deployment without the `backupnoconfig` sidecar upon `automatedbackup` CRD installation. 
- as you suggested, we can add annotation that disables the sidecar 

I vote for the second approach, to avoid rolling out a new revision